k8s/demo/common/bsp/identity.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: bsp
+  namespace: bsp
+spec:
+  type: 0
+  ResourceID: "/subscriptions/d025fece-ce99-4df2-b7a9-b649d3ff2060/resourceGroups/managed-identities-demo-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/bsp-demo-mi"
+  ClientID: "933dd8c6-3684-4fca-8985-d035acafd476"
+
+---
+
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentityBinding
+metadata:
+  name: bsp-binding
+  namespace: bsp
+spec:
+  AzureIdentity: bsp
+  Selector: bsp
```

Comment:
was this generated by script provided in bin folder?

Reply:
no, script  does not have demo, copied from portal.

Reply:
oh is it not? sad